src/Mgate/StatBundle/Controller/IndicateursController.php

```diff
@@ -955,48 +949,27 @@ private function getCA()
             }
         }
 
-        // Chart
         $series = [];
         foreach ($mandats as $idMandat => $data) {
-            //if($idMandat>=4)
-            $series[] = ['name' => 'Mandat ' . $etudeManager->mandatToString($idMandat), 'data' => $data];
+            $series[] = ['name' => 'Mandat ' . $idMandat, 'data' => $data];
         }
 
-        $style = ['color' => '#000000', 'fontWeight' => 'bold', 'fontSize' => '16px'];
-
-        $ob = new Highchart();
-        $ob->global->useUTC(false);
+        $chartFactory = $this->container->get('Mgate_stat.chart_factory');
```

Comment:
I don't really figure out what changed here, but the previous display was very well for us.

For instance the legend was `Mandat 2017/2018` this PR rolls it back to mandat 40, which is much more annoying.

Reply:
Apart from that, it's fine.

We really have to set how mandat field is used.

As far as I know, you are the only Junior using mandat (perhaps because you are much younger than many other Juniors) this way (as a counter since creation date). Would a migration on your side be possible to match the common system ?